tests/backup/backup_delete_test.go

```diff
@@ -74,9 +75,20 @@ var _ = Describe("{IssueDeleteOfIncrementalBackupsAndRestore}", func() {
 				cloudCredUidList = append(cloudCredUidList, cloudCredUID)
 				backupLocationUID = uuid.New()
 				credName = fmt.Sprintf("autogenerated-cred-%v", time.Now().Unix())
-				err := CreateCloudCredential(provider, CredName, CloudCredUID, orgID, ctx)
-				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of cloud credential named [%s] for org [%s] with [%s] as provider", CredName, orgID, provider))
+				err := CreateCloudCredential(provider, credName, cloudCredUID, orgID, ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of cloud credential named [%s] for org [%s] with [%s] as provider", credName, orgID, provider))
 				log.InfoD("Created Cloud Credentials with name - %s", credName)
+				cloudCredCreateStatus := func() (interface{}, bool, error) {
+					status, err := IsCloudCredPresent(credName, ctx, orgID)
+					if err != nil {
+						return "", true, fmt.Errorf("cloud cred %s is not created with error %v", credName, err)
+					}
+					if status {
```

Comment:
nit: Variable name `status` is not representation of something that is boolean; 
maybe use `ok` or `isPresent` or `cloudCredPresent`.